brightlayer-ui/react-native-workflows
# Bug/120 change password header

Fixes #120 .


#### Changes proposed in this Pull Request:
- Add a header with back button to change password dialog
- Clean up status bar styles
- Place SafeAreaProvider at the top instead of just around the preauth container


#### Screenshots:
Before:
![image](https://user-images.githubusercontent.com/29152776/152813558-169a20b1-7b1b-4f38-b6d0-b2ff8e7350da.png)

After:
![image](https://user-images.githubusercontent.com/29152776/152813331-2950e721-2cfc-4954-a34c-dcfdea0c6cad.png)


#### To Test:
- Run the demo project
- Open the change password dialog
- Observe that it has a title and the back button closes the dialog
